cfme/tests/webui/test_general_ui_manual.py

```diff
@@ -112,3 +112,106 @@ def test_pdf_summary_provider(provider):
         1503213
     """
     pass
+
+
+@pytest.mark.manual('manualonly')
+@pytest.mark.tier(1)
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.meta(coverage=[1740131])
+def test_red_hat_cloud_page_internal_server_error():
```

Comment:
@jawatts might want to remove this if your automation ends up covering it? Just a heads up for visibility.